### PR TITLE
Fix/image path

### PR DESCRIPTION
**Pull request by vyvydkf628**

### CODE REVIEW

**Comment by minsulee2** [comment]
Please resolve conflict

**Comment by vyvydkf628** [comment]
I have added additional commits.

File: src/components/ui/Answers.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import styled from 'styled-components'
-import {FiCheckCircle} from 'react-icons/fi'
-import {FaRegTimesCircle} from 'react-icons/fa'
+import styled from 'styled-components';
+import {FaRegTimesCircle, FaRegCheckCircle} from 'react-icons/fa';
```
Comment (minsulee2) [inline src/components/ui/Answers.js:4]: {  } space please

File: src/components/ui/ShareLink.js [hunk 2]
```diff
@@ -47,44 +59,46 @@ class ShareLink extends React.Component {
     };
 
     render(){
+        const {href} = this.state;
```
Comment (minsulee2) [inline src/components/ui/ShareLink.js:62]: { } space please

File: src/components/ui/ShareLink.js [hunk 3]
```diff
@@ -24,20 +26,30 @@ const Wrapper = styled.div`
...
         super();
 
         this.state = {
             clicked: false,
+            href: '',
         };
     }
 
+    componentDidMount = () => {
+        this.setState({href: window.location.href});
```
Comment (minsulee2) [inline src/components/ui/ShareLink.js:50]: { } space please

File: src/components/ui/Spinner.js [hunk 4]
```diff
@@ -34,15 +34,15 @@ class Spinner extends React.Component {
     }
 
     render() {
-        const style = {position: "fixed", top: "50%", left: "50%", transform: "translate(-50%, -50%)"};
+        const style = {position: 'fixed', top: '50%', left: '50%', transform: 'translate(-50%, -50%)'};
```
Comment (minsulee2) [inline src/components/ui/Spinner.js:37]: Why don't you put them below directly?

File: src/components/ui/Answers.js (hunk 1 above)
Comment (vyvydkf628) [reply src/components/ui/Answers.js]: Done!

File: src/components/ui/ShareLink.js (hunk 2 above)
Comment (vyvydkf628) [reply src/components/ui/ShareLink.js]: Done!

File: src/components/ui/Spinner.js (hunk 4 above)
Comment (vyvydkf628) [reply src/components/ui/Spinner.js]: Done!

File: src/components/ui/ShareLink.js (hunk 3 above)
Comment (vyvydkf628) [reply src/components/ui/ShareLink.js]: Done!

**Pull request merged**